buildenv/jenkins/parentsJobPipeline.groovy

```diff
@@ -0,0 +1,88 @@
+#!groovy
+def javaVersions = params.javaVersions.trim().split("\\s*,\\s*");
+def buildLists = params.buildLists.trim().split("\\s*,\\s*");
```

Comment:
Should we use the same parameter names as testJobTemplate? i.e., javaVersions -> JDK_VERSIONS, buildLists -> GROUPS

Reply:
Yes, definitely. The reason I'm using the javaVersions is that will be consistent with upstream naming convention as the configuration files are from upstreams.

Reply:
Updated.